google_takeout_parser/parse_json.py

```diff
@@ -137,9 +138,34 @@ def _parse_location_history(p: Path) -> Iterator[Res[Location]]:
         except Exception as e:
             yield e
 
-
 _parse_location_history.return_type = Location  # type: ignore[attr-defined]
 
+def _parse_semantic_location_history(p: Path) -> Iterator[Res[Location]]:
+    json_data = json.loads(p.read_text())
+    if "timelineObjects" not in json_data:
+        yield RuntimeError(f"Locations: no 'timelineObjects' key in '{p}'")
+    timelineObjects = json_data.get("timelineObjects", [])    
+    print(f"timelineObjects length:  {len(timelineObjects)}")
+    for timelineObject in timelineObjects:
+        if "placeVisit" in timelineObject:
+            placeVisit = timelineObject.get("placeVisit")
+            visitConfidence = placeVisit.get("visitConfidence")
+            location = placeVisit["location"]
```

Comment:
This is just a JSON dict here I believe, so would need to be parsed. But see my comment above about placing this data onto the `PlaceVisit` directly